QueryBuilder/Compilers/SnowflakeCompiler.cs

```diff
@@ -42,6 +42,31 @@ public override string Parameter(SqlResult ctx, object parameter)
             }
         }
 
+        protected override string CompileColumns(SqlResult ctx)
+        {
+            /**
+             * Snowflake supports the ANY_VALUE function natively
+             */
+            ctx.Query.Clauses = ctx.Query.Clauses.Select(clause =>
+            {
+                if (clause is AggregateAnyValueColumn column)
```

Comment:
Why? `AggregateAnyValueColumn ` implements `Compile`, isn't that enough?

Reply:
The base `AggregateAnyValueColumn` implements a generic 'any value' by generating a `MIN(...)` clause, but for snowflake we want to use the optimized, snowflake specific, `ANY_VALUE()` function. Which is achieved here by the `AggregateGenericColumn` with a `Type` of `any_value` (it's compile will then generate `ANY_VALUE(...)`).